sources/packages/web/src/components/common/ManageProgramAndOfferingSummary.vue

```diff
@@ -1,39 +1,28 @@
 <template>
-  <div>
-    <program-offering-detail-header
-      class="m-4"
-      :headerDetails="{
-        ...educationProgram,
-        institutionId: institutionId,
-        status: educationProgram.programStatus,
-      }"
-    />
-    <v-card>
-      <v-container>
-        <ProgramDetails
-          :programId="programId"
-          :locationId="locationId"
-          :educationProgram="educationProgram"
-        />
-        <hr class="horizontal-divider" />
-        <OfferingSummary
-          :programId="programId"
-          :locationId="locationId"
-          :institutionId="institutionId"
-        />
-      </v-container>
-    </v-card>
-  </div>
+  <v-card>
```

Comment:
Do we need a v-card and v-container in this component as full page container will be wrapping it?

Reply:
yes, we need it in the component. the `view/` page is a `centered` layout, which doesn't have a card. 
![image](https://user-images.githubusercontent.com/77353155/188738437-37e292a6-f2b5-4cb7-b776-90f4dd0f90c7.png)